NineChronicles.Headless.Executable/Commands/ReplayCommand.cs

```diff
@@ -378,19 +382,41 @@ public void Help([FromService] ICoconaHelpMessageBuilder helpMessageBuilder)
 
             var block = GetBlockData(graphQlClient, transactionResult.BlockHash)?.ChainQuery?.BlockQuery?.Block;
             var previousBlockHashValue = block?.PreviousBlock?.Hash;
+            var previousBlockHashStateRootHash = block?.PreviousBlock?.StateRootHash;
             var preEvaluationHashValue = block?.PreEvaluationHash;
             var minerValue = block?.Miner;
-            if (previousBlockHashValue is null || preEvaluationHashValue is null || minerValue is null)
+            if (previousBlockHashValue is null || preEvaluationHashValue is null || minerValue is null || previousBlockHashStateRootHash is null)
             {
                 throw new CommandExitedException("Failed to get block from query", -1);
             }
             var miner = new Address(minerValue);
 
-            var explorerEndpoint = $"{endpoint}/explorer";
```

Comment:
So the explorer endpoint will be removed after this pr is merged?

Reply:
This seems to be the case when using the replay command. I'm not sure if I understand it right, but the /explorer endpoint on headless will be preserved.

Reply:
Ah i understand, it's just for the replay command 👍